contracts/infrastructure/ens/ArgentENSManager.sol

```diff
@@ -125,7 +125,7 @@ contract ArgentENSManager is IENSManager, Owned, Managed {
      * @param _subnode The target subnode.
      * @return true if the subnode is available.
      */
-    function isAvailable(bytes32 _subnode) public view returns (bool) {
+    function isAvailable(bytes32 _subnode) external view returns (bool) {
```

Comment:
Is this change necessary? I thought the idea was to not touch the infrastructure contracts for the next release.

Reply:
These were raised by `slither` on the ENS contracts for [this rule](https://github.com/crytic/slither/wiki/Detector-Documentation#public-function-that-could-be-declared-as-external). They are not dangerous so have ignored them for now but good to fix in future.